<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPlainTextEdit" name="plainTextEdit"/>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="decryptfile_pushButton">
       <property name="text">
        <string>解密文本</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="encryptfile_pushButton">
       <property name="text">
        <string>加密文本</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="encrypt_pushButton">
       <property name="text">
        <string>加密字符串</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="decrypt_pushButton">
       <property name="text">
        <string>解密字符串</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
